### PR TITLE
OEPCIS-603: fixed the code to provide timestamp precision of millisecond

### PR DESCRIPTION
**Pull request by DevangChauhan**
created a PR for code changes to provide timestamp precision of millisecond. Kindly please review and merge this PR.

### CODE REVIEW

**Review by sboeckelmann — CHANGES_REQUESTED** [review]
also truncate to millis in CustomInstantAdapter.java

**Comment by DevangChauhan** [comment]
added a truncate to Millis in CustomInstantAdapter.java

File: epcis/src/main/java/io/openepcis/model/epcis/modifier/CustomInstantAdapter.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import java.time.ZonedDateTime;
 import java.time.format.DateTimeFormatter;
 import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoUnit;
 
 public class CustomInstantAdapter extends XmlAdapter<String, OffsetDateTime> {
```
Comment (sboeckelmann) [inline epcis/src/main/java/io/openepcis/model/epcis/modifier/CustomInstantAdapter.java:26]: missed out on marshal method

**Review by sboeckelmann — CHANGES_REQUESTED** [review]

File: epcis/src/main/java/io/openepcis/model/epcis/modifier/CustomInstantAdapter.java (hunk 1 above)
Comment (DevangChauhan) [reply epcis/src/main/java/io/openepcis/model/epcis/modifier/CustomInstantAdapter.java:26]: I have added truncate to millis in the marshal method also.

**Review by sboeckelmann — APPROVED** [review]

**Pull request merged**